tf_glue_barebones.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from dataclasses import dataclass, field
+
+from data_processors import DataProcessorForSequenceClassification
+from tf_training_args import TFTrainingArguments
+from trainer_tf_barebones import TFTrainer
+from transformers import AutoConfig, AutoTokenizer
+from transformers import TFAutoModelForSequenceClassification
+
+
+logging.basicConfig(level=logging.INFO)
+
+
+
+@dataclass
+class ModelArguments:
+    model_name_or_path: str = field(
+        metadata={"help": "Path to pretrained model or model identifier from huggingface.co/models"}
+    )
+
+
+@dataclass
+class DataTrainingArguments:
+    task_name: str = field(metadata={"help": "The name of the task to train on"})
+    data_dir: str = field(
+        metadata={"help": "The input data dir. Should contain the .tsv files (or other data files) for the task."}
+    )
+    max_seq_length: int = field(
+        default=128,
+        metadata={
+            "help": "The maximum total input sequence length after tokenization. Sequences longer "
+            "than this will be truncated, sequences shorter will be padded."
+        },
+    )
+
+
+# TODO(Parse from nested JSON file if necessary)
+
+model_args = ModelArguments(model_name_or_path="bert-base-cased")
+data_args = DataTrainingArguments(task_name="glue/mrpc", data_dir=".", max_seq_length=128)
+training_args = TFTrainingArguments(output_dir="./save")
+
+
+
+tokenizer = AutoTokenizer.from_pretrained(model_args.model_name_or_path, use_fast=True)
+
+
+# This DataProcessorForSequenceClassification will not be needed in the future b/c Thomwolf's
+# datasets will do that.
+processor = DataProcessorForSequenceClassification(
+    **vars(data_args), 
+    dataset_name="glue/mrpc",
+    format="tfds",
+    guid="idx",
+    text_a="sentence1",
+    text_b="sentence2",
+    label="label",
+)
```

Comment:
I fully agree. This part (data processor) needs to be way much simplified with Thom's nlp lib.